BLSQ/iaso
# IA-1597, IA-2412: move components to bluesquare-components

Generic components can be moved out of iaso

Related JIRA tickets : IA-1597, IA-2412

## Self proofreading checklist


## Doc

N/A

## Changes

⚠️ Requires sister PR on bluesquare-components: https://github.com/BLSQ/bluesquare-components/pull/170

IA-1597: TextArea:
- Delete component
- Add type `"textarea"` to `InputComponent`
- Replace `TextArea` with `InputComponent` everywhere, excpet `MultilineText`, which itself wraps `TextArea`
- Update imports: 
    - ReviewOrgUnitChangesCommentDialog
    -  StatusModal (storages)
    - CreateBudgetStep
    - CreateBudegtStepOverride
    - VaccineRequestForm

IA-2412: InputWithInfos
- Delete component
- Update imports:
    - CompletenessStatsFilters
    - InstancesFiltersComponent
    - OrgUnitsFilters
    - OrgUnitTypesDialog
    - OrgUnitWriteTypes
    - UserOrgUnitWriteTypes
    - UsersInfos
    - ChronogramTaskCreateEditModal
    - ChronogramTelmplateTAskCreateEditModal
    - CreateEditReasonForDelay
    - lasAfroMapFilters

## How to test

update package.json to use IA-1597_add_TextArea from bluesquare-components and run `npm update bluesquare-components`

Go to the pages impacted by the refactor, nothing should have changed

## Print screen / video

Textarea in Budget tool

![Screenshot 2024-12-23 at 15 37 30](https://github.com/user-attachments/assets/cefe8180-af40-44fb-9af7-a435a0d53405)

 InputWithInfos


https://github.com/user-attachments/assets/8c1f10f7-f8cb-4dbb-80d5-814cdf5e8eb2




## Notes
REQUIRES https://github.com/BLSQ/bluesquare-components/pull/170

Failing JS tests will pass once the Pr on bluesquare-components are fixed

## Follow the Conventional Commits specification

The **merge message** of a pull request must follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) specification.

This convention helps to automatically generate release notes.

Use lowercase for consistency.

[Example](https://github.com/BLSQ/iaso/commit/8b8d7d3064138c1e57878f17b4eb922516ab0112):

```
fix: empty instance pop up

Refs: IA-3665
```

Note that the Jira reference is preceded by a _line break_.

Both the line break and the Jira reference are entered in the _Add an optional extended description…_ field.
